Vic2ToHoI4/Source/HOI4World/HoI4World.h

```diff
@@ -103,6 +103,7 @@ class World: commonItems::parser
 	[[nodiscard]] const auto& getGenericFocusTree() const { return genericFocusTree; }
 	[[nodiscard]] const auto& getLocalisation() const { return *hoi4Localisations; }
 	[[nodiscard]] const auto& getSoundEffects() const { return soundEffects; }
+	[[nodiscard]] const auto& getDate() const { return theDate; }
```

Comment:
```
return *theDate;
```

No need to return the unique_ptr, when what's really wanted is (a reference to) the underlying item.